.vscode/extensions.json

```diff
@@ -0,0 +1,10 @@
+{
+  "recommendations": [
+    "esbenp.prettier-vscode",
+    "dbaeumer.vscode-eslint",
+    "github.vscode-pull-request-github",
+    "eamodio.gitlens",
+    "christian-kohler.path-intellisense",
+    "stylelint.vscode-stylelint"
+  ]
+}
```

Comment:
@ivanms1  
This is really helpful! I didn't know about this feature of VS Code. Thanks for adding it! When I set up the project in VS Code, I didn't realize there were recommended extensions. Do you think it's a good idea to mention it in the `README` as well?

<img width="1318" alt="image" src="https://user-images.githubusercontent.com/24983797/204120620-3c63fa2a-7698-48e7-a026-8ff9c03865be.png">

Reply:
oh yeah, I'll add it in the readme.